src/fracTrackerMobileLayer.js

```diff
@@ -9,8 +9,13 @@ L.GeoJSON.FracTrackerMobile = L.GeoJSON.extend(
     },
 
     onAdd: function(map) {
-      map.on('moveend', this.requestData, this);
+      var info = require('./info.json');
```

Comment:
These lines of code check the map view with the layer's extents before requesting data. We already had a layer that does this but it is a little different.
https://github.com/publiclab/leaflet-environmental-layers/blob/9b463c24f639230f359be445dd6b036d88d148c0/src/osmLandfillMineQuarryLayer.js#L45-L48

This seems to be cleaner but the reason I chose to proceed with what I had was because,
Let's say we have a layer displayed on the map but it is outside the layer's extents. Here the layer will no longer request data on map move. We want the data to be requested when the layer is turned on and off which is not possible in the code from the osm file.

Reply:
I shall post the link to gh-pages once it is updated. I haven't changed the osm layer so the difference I had mentioned above can be tested out using the link.